gma/integration_test/src/integration_test.cc

```diff
@@ -340,6 +350,28 @@ void FirebaseGmaPreInitializationTests::SetUpTestSuite() {
 
 // Test cases below.
 
+TEST_F(FirebaseGmaMinimalTest, TestInitializeGmaWithoutFirebase) {
+  // TODO(almostmatt): Disabling mediation initialization to not mess up next test on iOS. 
+  firebase::gma::DisableMediationInitialization();
+  LogDebug("Initializing GMA without a Firebase App.");
+  firebase::InitResult result;
+#if defined(ANDROID)
+  ::firebase::gma::Initialize(app_framework::GetJniEnv(),
+                              app_framework::GetActivity(), &result);
+#else   // !defined(ANDROID)
+  ::firebase::gma::Initialize(&result);
+#endif  // defined(ANDROID)
+  EXPECT_EQ(result, ::firebase::kInitResultSuccess);
+  LogDebug("Successfully initialized GMA.");
+  // Workaround: GMA does some of its initialization in the main
+  // thread, so if you terminate it too quickly after initialization
+  // it can cause issues.  Add a small delay here in case most of the
+  // tests are skipped.
+  ProcessEvents(1000);
```

Comment:
Is it cleaner to wait on the Future that Initialize returns?

Reply:
Yea, I updated this and the main testsuite teardown to wait for initialization futures instead of an arbitrary 1 second duration.